examples/tv-app/tv-common/clusters/messages/MessagesManager.h

```diff
@@ -21,6 +21,115 @@
 
 #include <iostream>
 #include <list>
+#include <vector>
+
+struct CachedMessageOption
+{
+    constexpr static size_t kResponseIdMin  = 1;
+    constexpr static size_t kLabelMaxLength = 32;
+
+    CachedMessageOption(uint32_t id, std::string label) :
+        mLabel(label), mOption{ chip::MakeOptional(id), chip::MakeOptional(chip::CharSpan::fromCharString(mLabel.c_str())) }
+    {}
+
+    CachedMessageOption(const CachedMessageOption & option) :
+        mLabel(option.mLabel),
+        mOption{ option.mOption.messageResponseID, chip::MakeOptional(chip::CharSpan::fromCharString(mLabel.c_str())) } {};
+
+    CachedMessageOption & operator=(const CachedMessageOption & option) { return *this; };
```

Comment:
This does not look like a valid assignment operator... what is going on here?

Reply:
I would expect an impl like:

```cpp
mLabel = option.mLabel;
mOption.messageResponseID = option.mOption.messageResponseID;
mOption.label = chip::MakeOptional(chip::CharSpan::fromCharString(mLabel.c_str()));
```

or a `= delete`

Reply:
removed this operator since its currently noop

Reply:
marked the assignment operator as delete